### ISSUE TITLE
[Win10]1Password Crashes v0.10.3

### ISSUE THREAD

**Issue by duff2481** [brave/browser-laptop#2280]
When trying to use 1Password, brave crashes every time 

**Expected behavior:**
Filling out password of course ;-) 
- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 
- Brave Version:
v.0.10.3
- Steps to reproduce:
    1. I have visited multiple site and each time same result.  Brave unexpectedly crashes.

**Comment by luixxiul** [comment]
Do you experience the issue on the latest release?

**Comment by duff2481** [comment]
Yes, however i have to right click and select 1password for it to come up.
I'm unable to use 'ctrl \' but this may be an issue with 1password not
Brave.

Perhaps I can use Brave as daily driver again!

**Comment by bsclifton** [comment]
@duff2481 1Password just released a major update this morning, which including recognizing the signature for Brave. Can you please update to latest and let us know how it works? The Ctrl \ works great for me :smile:

**Comment by luixxiul** [comment]
I'm going to close the issue in favor of https://community.brave.com/t/1password-extension/5104/7?u=suguru

@duff2481 If you are still around here, would you please join the conversation there to talk about the issue further? :smile: There is also a lot of discussions on the community, and you would be able to find solutions for other issues you might experience :+1: Thanks!
